Makefile

```
@@ -174,3 +178,51 @@ watch-ts: ## watch changes in TypeScript files
 help:
 	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 .PHONY: help
+
+###########################################
+# Translations tasks
+
+.PHONY: i18n-generate
+i18n-generate: ## Generate source translations files for all applications
```

I think it's more standard to define called rules as rule dependencies.

```suggestion
i18n-generate: i18n-generate-back i18n-generate-front  ## Generate source translations files for all applications
```

---

I will do it in an other PR, I made an issue (#578)